src/test/suite/participant/participant.test.ts

```diff
@@ -508,7 +508,7 @@ suite('Participant Controller Test Suite', function () {
             };
             await invokeChatHandler(chatRequestMock);
             const messages = sendRequestStub.secondCall.args[0];
-            expect(messages[0].content).to.include(
+            expect(messages[1].content).to.include(
```

Comment:
What is now `messages[0]`?

Reply:
`message[0]` is the assistant message, `message[1]` is the user message. 

In this pr we move the namespace, schema, and sample document passing to the user prompt. I'm thinking these should be in the user prompt as we don't want any of the information from the data passed to impact what the ai will respond with. This is similar to a system prompt, if you allow for unsanitized text to make it's way into the system prompt then it can be susceptible to things like prompt injection.